### DISCUSSION TITLE
support for XBOX controller

### DISCUSSION THREAD

**Discussion by privatepenguinzero** [mgsweet/Apex-NoRecoil-2021#306] [Ideas]
Is it possible to make the AHK code work on controller button press instead of mouse? if yes, how?
Thank you in advance

**Comment by RaphielHS** [comment]
possible by changing the source code to support controller inputs
https://www.autohotkey.com/docs/v1/KeyList.htm#Joystick

you can play around and test stuff out

but I doubt it would work because how the code sends the move command

        DllCall("mouse_event", uint, 0x01, uint, Round(x * modifier), uint, Round(y * modifier))

don't know if the code for moving the cursor will work with controller but you can find out of you want to

**you can always google stuff and trying them first**
